setup.cfg

```diff
@@ -1,6 +1,6 @@
 [flake8]
 exclude = */migrations/*,.env/*
-ignore = E226,E261
+ignore = E226,E261,F403
```

Comment:
Instead of ignoring these warnings globally, you should add `# noqa` wherever you have `from something import *`